src/irmin-pack/unix/gc_worker.ml

```diff
@@ -359,17 +376,28 @@ module Make (Args : Gc_args.S) = struct
       | `Delete -> None
       | `Archive lower ->
           [%log.debug "GC: archiving into lower"];
-          (* off should not include old dead bytes *)
-          let off = Dispatcher.suffix_start_offset dispatcher in
+          stats :=
+            Gc_stats.Worker.finish_current_step !stats "archive: iter reachable";
+          let discarded_data = ref Live.empty in
```

Comment:
```suggestion
          let to_archive = ref Live.empty in
```

Or something that indicates it is the data that is kept by archiving to the lower. (This proposed name is just based on the input argument for `Lower.archive_seq_exn`) The current name makes me think this would hold the orphaned data (but the `Live.empty` helps me understand it is not).

Reply:
Yep, thanks!